### PR TITLE
Google signin

### PR DESCRIPTION
**Pull request by matallo**
/cc @juanignaciosl @xavijam need your :eyes: here

to be deployed with https://github.com/CartoDB/cartodb-central/pull/411

### CODE REVIEW

**Comment by xavijam** [comment]
I've checked that google sign in redirects to username.cartodb.com/u/username, so it gets a 404 error, because that user doesn't exist. cc/@juanignaciosl

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1347/))

**Comment by matallo** [comment]
@xavijam yeah, already spotted this and alerted @juanignaciosl can you take a look?

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1364/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1368/))

**Comment by juanignaciosl** [comment]
@xavijam take a look [at my last commit](https://github.com/CartoDB/cartodb/commit/b47d1b2e250d6c23d17d37b720e3e4f190248c98). It's a old fix of mine in this same branch I rolled back weeks ago because it felt *really* strange to me that it was wrong. As you see it's not Google related at all. I know it's related to subdomains and organisations. In fact without that change not even normal (not Google) login redirection works. Please try again and check those lines.

**Comment by xavijam** [comment]
Checking it...

**Comment by xavijam** [comment]
@matallo:
- [ ] Remove login button in user domain login (i feel it doesn't make sense...)
- [ ] Input placeholders style are not different as the value ones (italic?, opacity?)

cc @saleiva

**Comment by juanignaciosl** [comment]
@xavijam why would login button in user domain not make sense? If user has not set a password he could not even be able to login.

**Comment by xavijam** [comment]
screen shot 2015-02-02 at 12 40 03

**Comment by saleiva** [comment]
Yeah, we can remove that link
+1 on the placeholders thing too

**Comment by juanignaciosl** [comment]
@rafatower could you CR this, please? Same comment as Central PR.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1369/))

File: config/application.rb [hunk 1]
```diff
@@ -110,6 +112,7 @@ class Application < Rails::Application
       specs.css
       table.css
       tables.css
+      frontend
```
Comment (rafatower) [inline config/application.rb]: without extension?

**Comment by rafatower** [comment]
:+1:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1388/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1395/))

File: config/application.rb (hunk 1 above)
Comment (matallo) [inline config/application.rb]: actually the extension is not needed as it's `frontend.scss` the other stylesheets are `tables.css.scss` thus it seems there's no extension

**Comment by matallo** [comment]
:sparkles:

**Comment by xavijam** [comment]
:dancer:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1401/))

**Pull request merged**